sympy/physics/continuum_mechanics/beam.py

```diff
@@ -2177,18 +2188,18 @@ def draw(self, pictorial=True):
             >>> b.apply_load(10, 30, 1, 50)
             >>> b.apply_load(M, 15, -2)
             >>> b.apply_load(-M, 30, -2)
-            >>> b.apply_support(50, "pin")
-            >>> b.apply_support(0, "fixed")
-            >>> b.apply_support(20, "roller")
+            >>> p50 = b.apply_support(50, "pin")
+            >>> p0, m0 = b.apply_support(0, "fixed")
+            >>> p20 = b.apply_support(20, "roller")
             >>> p = b.draw()  # doctest: +SKIP
-            >>> p  # doctest: +ELLIPSIS
+            >>> p  # doctest: +SKIP
             Plot object containing:
             [0]: cartesian line: 25*SingularityFunction(x, 5, 0) - 25*SingularityFunction(x, 23, 0)
             + SingularityFunction(x, 30, 1) - 20*SingularityFunction(x, 50, 0)
             - SingularityFunction(x, 50, 1) + 5 for x over (0.0, 50.0)
             [1]: cartesian line: 5 for x over (0.0, 50.0)
             ...
-            >>> p.show()
+            >>> p.show() # doctest: +SKIP
```

Comment:
Would this disable the plot showing in the docs? If so, I don't think we want to do that.

Reply:
I hadn't thought about that, thank you for pointing it out. That is indeed not what we want. When I run the doctest I get a NameError that 'p' is not defined because it is skipped in line 2195. This was my solution for this problem, but as you pointed out this is not wanted. 
If I remove the `#doctest: +SKIP`  from the docstring the Error gets replaced with a UserWarning. Which is why I think it was skipped in the first place. 
Do you have a suggestion on how to handle this problem?

Reply:
When I build the docs locally it builds as expected:

![image](https://github.com/sympy/sympy/assets/276007/aecc5596-b5df-4f86-b122-b490c9d28723)